ml-agents/mlagents/trainers/tf_policy.py

```diff
@@ -88,13 +96,23 @@ def __init__(self, seed, brain, trainer_parameters):
                     "The memory size for brain {0} is 0 even "
                     "though the trainer uses recurrent.".format(brain.brain_name)
                 )
-            elif self.m_size % 4 != 0:
-                raise UnityPolicyException(
```

Comment:
Just to be sure, have you tested lstm with a prime number of memories ?

Reply:
I added this back in. It turns out even on `master` the check wasn't sufficient to guard SAC from m_size not divisible by 8... but now it should be correct.